modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -195,24 +158,23 @@ class EventDispatcher<T> {
 
         // Unbind handlers marked with "once"
         if (callback.once) {
-          this.off(name, callback.func);
+          this.off(lcName, callback.func);
         }
 
         // Stop immediate propagation if needed
-        if (args.isImmediatePropagationStopped()) {
-          args.stopPropagation();
```

Comment:
Note: This was intentionally removed, as it's no longer needed since the normalized function calls this immediately.